packaging/npm-publish.sh

```diff
@@ -21,6 +21,7 @@ $NIMIQ_VERSION
     "type": "git",
     "url": "https://github.com/nimiq-network/core.git"
   },
+  "main": "web.esm.js",
```

Comment:
The thing is, `web.js` is not a module of any kind I believe? It's not a commonjs or umd or any build. I don't think it would work as a `main` entry point?